### DISCUSSION TITLE
Linux/Alsa MIDI?

### DISCUSSION THREAD

**Discussion by nathan-stewart** [RomanKubiak/ctrlr#354] [Q&A]
I know this is an older project but it looks like there's some recentish discussion.  I found a 9 yo thread referencing an older thread which I couldn't find about it, but I'm trying to get the AMT8 and JV-880 panels working in standalone, and I see no MIDI devices. Even the MIDI settings menu is grayed out.  Amidi --list-devices shows them, and I can use them from Ardour:

Dir Device    Name
IO  hw:2,0,0  UMC1820 MIDI 1
IO  hw:3,0,0  Unitor8 MIDI 1
...
hw:3,0,7  Unitor8 MIDI 8
IO  hw:3,0,8  Unitor8 Broadcast

My Ctrlr app is:
Version = 6.0.35, Build date = Mon 11 May 2020 08:19:12 PM CEST, Branch = Stable, Juce = 5.4.1, libusb = 1.0.19, liblo = 0.28, lua = Lua 5.1.4, luabind = 0.9.0, boost = 1.67.0,

**Comment by dnaldoog** [comment]
I personally have no idea, but I would use only version _5.3.201_ on Linux and Windows! :)
